entity-service/src/main/resources/configs/common/application.conf

```diff
@@ -21,6 +21,28 @@ attribute.service.config = {
   port = ${?ATTRIBUTE_SERVICE_PORT_CONFIG}
 }
 
+entity.service.change.skip.attributes = [
+  {
+    scope = API
+    attributes = ["last_called_millis", "change_label_timestamp", "api_discovery_timestamp"]
+  },
+  {
+    scope = ACTOR
```

Comment:
Get rid of this common config. It should get applied from helm/values as needed by the use case